v3-docs/v3-migration-docs/index.md

```diff
@@ -15,15 +15,15 @@ Meteor 3.0 is currently in its Release Candidate (RC) phase, a nearly final vers
 **Node.js version:** `20.11.1 LTS` <br/>
 **NPM version:** `10.2.4`
 
-## How to test Meteor 3.0?
+## How to try Meteor 3.0?
```

Comment:
Changed it to `try` since `test` got me thinking it was about testing a Meteor 3 app until I read the actual command.